.circleci/config.yml

```diff
@@ -235,8 +235,13 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - news_update
-        
+      - news_update:
+          filters:
+            branches:
+              ignore: master
```

Comment:
Should this be `develop`?

Reply:
Nice catch ! stole that form Cyclus... forgot to change it !!